cypress/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["plugin:cypress/recommended"]
```

Comment:
Can you try to move it to the root config? Otherwise it won't be enforced by CI.